Release notes/Release.md

```diff
@@ -0,0 +1,243 @@
+## 22 March 2023
+
+<!DOCTYPE html>
+<html>
+<body>
+<table>
+  <tr style="background:none;"> 
+     <td><div style=text-align:left;>
+     <img src="./releasenotesicon/viewer.png" style="width: 40px; height: 40px;"></div></td><td><h3><b>Viewer role</b><h3></td>
+  </tr>
+  </table>
+</body>
+</html>
+
+- As a viewer role in ADEX, users can browse topics and explore the features of the portal, but they are not able to perform any actions in the portal. This means that they cannot make any changes to the data in the portal.
```

Comment:
- A new Viewer role has been introduced. A user with Viewer role can browse topics and explore the portal, but is unable to make any changes.
- A user can be assigned with a Viewer role within a team. This means the team can have Viewer users to only see project and topic details safely.

Reply:
added KC